### PR TITLE
sponsors

### PR DESCRIPTION
**Pull request by ArshitaKalra**
I have added the title sponsors !

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Can you post the screenshot of both! Mobile and Desktop views @ArshitaKalra ?

File: css/style.css [hunk 1]
```diff
@@ -471,6 +472,53 @@ header .intro-text {
...
+	color: #888;
+}
+#spon p {
+	color: #f6f6f6;
+}
+#spon .service-desc {
+	margin-bottom: 40px;
+}
+#spon .team-img1{
+	width: 400px;
```
Comment (shreyasbapat) [inline css/style.css]: Can you make these relative? Like in percentages?

File: css/style.css [hunk 2]
```diff
@@ -471,6 +472,53 @@ header .intro-text {
...
+	height: 300px;
```
Comment (shreyasbapat) [inline css/style.css]: This too!

File: css/style.css [hunk 3]
```diff
@@ -471,6 +472,53 @@ header .intro-text {
...
+	margin-left: 0px;
+}
+#spon .team-img2{
+	width: 380px;
```
Comment (shreyasbapat) [inline css/style.css]: Similarly this! 
Absolute values are non mobile friendly!

**Comment by ArshitaKalra** [comment]
please check the changes @shreyasbapat

**Comment by shreyasbapat** [comment]
@ArshitaKalra I took the liberty to commit one extra change! Thanks for the PR!

**Pull request merged**